src/components/bestCities/__tests__/BestCityCard.test.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { render, cleanup } from '@testing-library/react';
+import { MemoryRouter as Router } from 'react-router-dom';
+import BestCityCard from '../BestCityCard';
+import CityContext from '../../../contexts/CityContext';
+
+afterEach(cleanup);
+
+it('should render CityCard', () => {
+  let cityData = '';
```

Comment:
Replace `cityData` with `cities` and the tests will pass.  You are passing `cities ={title}` not `cityData `that is why it comes as` undefined`